projects/plugins/jetpack/modules/stats/class-jetpack-stats-upgrade-nudges.php

```diff
@@ -178,38 +326,51 @@ private static function print_item( $title, $icon, $subitem = false, $button_lab
 	 */
 	public static function render( $callback ) {
 		/** This filter is documented in _inc/lib/admin-pages/class.jetpack-react-page.php */
-		if ( 'stats_reports_page' !== $callback || ! apply_filters( 'jetpack_show_promotions', true ) ) {
+		if ( 'stats_reports_page' !== $callback || ! apply_filters( 'jetpack_show_promotions', true ) || ! current_user_can( 'manage_options' ) ) {
 			return;
 		}
 
 		if ( ! defined( 'JETPACK_DEV_TEST_STATS_UPGRADE_NUDGES' ) || ! JETPACK_DEV_TEST_STATS_UPGRADE_NUDGES ) {
 			return;
 		}
 
+		if ( self::has_complete_plan() ) {
+			return;
+		}
+
+		if (
+			self::has_security_plan() &&
+			self::is_backup_active() &&
+			self::is_scan_active() &&
+			self::is_akismet_active() &&
+			self::is_search_active() &&
+			self::is_backup_active() &&
+			self::is_backup_active() &&
+			self::get_boost_output( false ) &&
+			self::get_crm_output( false )
```

Comment:
It looks like we already check `is_backup_active` in the second line of the if conditions. Also, we want to check that `get_boost_output` and `get_crm_output` return false.

```suggestion
			! self::get_boost_output( false ) &&
			! self::get_crm_output( false )
```